Dockerfile

```diff
@@ -0,0 +1,11 @@
+# APP
+FROM node:12
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+COPY src/ src/
+
+ENTRYPOINT ["node", "./src/index.js"]
```

Comment:
entrypoint file missing?

best thing is pull the project and make sure you can
1. npm install
2. build docker
3. run successfully